common/src/stack/report-system/command/report/system/__init__.py

```diff
@@ -63,16 +63,16 @@ def run(self, params, args):
 
 		# make it real ugly.
 		if exitonfail and not pretty:
-			_return_code = main(['--verbose', '--exitfirst', *args, *tests])
+			_return_code = main(['--verbose', '--exitfirst', '-rs', *args, *tests])
```

Comment:
It looks like these extra arguments might have been added by mistake.

Reply:
I added this because pytest by default doesn't show error messages if a test is skipped. When port 68 is taken, I skip the test and display a message but without `-rs` it won't show the message saying port 68 is in use. Only a couple of other report system tests ever skip a test so it shouldn't clog the output really.

Reply:
If we are going to add this option, we probably should use a longoption if it is available. `--display-skipped` or something similar instead of `-rs`.

Reply:
Looks like this option is `-r` with `s` as an add on, with no longopt alternative:

```
  -r chars              show extra test summary info as specified by chars
                        (f)ailed, (E)error, (s)skipped, (x)failed, (X)passed,
                        (p)passed, (P)passed with output, (a)all except pP.
                        Warnings are displayed at all times except when
                        --disable-warnings is set
```

:(

@bgreenb could you just add a brief comment right above that says what it is for?

Reply:
@bsanders Done